addons/binding/org.openhab.binding.network/src/main/java/org/openhab/binding/network/internal/NetworkHandlerConfiguration.java

```diff
@@ -0,0 +1,24 @@
+/**
+ * Copyright (c) 2010-2017 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.network.internal;
+
+import java.math.BigDecimal;
+
+/**
+ * Contains the handler configuration
+ *
+ * @author David Graeff
+ */
+public class NetworkHandlerConfiguration {
+    public String hostname;
+    public BigDecimal port;
+    public BigDecimal retry;
+    public BigDecimal refresh_interval;
```

Comment:
Is this because of the name in the xml? Pitty

Reply:
We're breaking the interface anyway with the next release. Do u want me to rename this configuration?

Reply:
Let's do that